AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -202,6 +202,15 @@ - (void)willRemoveSubview:(UIView *)subview
 #endif
 }
 
+- (CGSize)sizeThatFits:(CGSize)size
+{
+  ASDisplayNode *node = _asyncdisplaykit_node; // Create strong reference to weak ivar.
```

Comment:
Sounds good, hadn't seen the class since it was updated to use the pattern - consistency is great since it would be fast for sure.